### ISSUE TITLE
Predicting words instead of probabilities?

### ISSUE THREAD

**Issue by aliabbasjp** [tensorflow/tensorflow#271]
The tensorflow tutorial on language model allows to compute the probability of sentences :

probabilities = tf.nn.softmax(logits)

in the comments below it also specifies a way of predicting the next word instead of probabilities but does not specify how this can be done. So how to output a word instead of probability using this example?
```
lstm = rnn_cell.BasicLSTMCell(lstm_size)
# Initial state of the LSTM memory.
state = tf.zeros([batch_size, lstm.state_size])

loss = 0.0
for current_batch_of_words in words_in_dataset:
    # The value of state is updated after processing each batch of words.
    output, state = lstm(current_batch_of_words, state)

    # The LSTM output can be used to make next word predictions
    logits = tf.matmul(output, softmax_w) + softmax_b
    probabilities = tf.nn.softmax(logits)
    loss += loss_function(probabilities, target_words)

```

**Comment by vrv** [comment]
Hi @aliabbasjp: This kind of question is better suited to the StackOverflow forums or the discuss@tensorflow.org mailing list -- can you please repost there?  Thanks!

**Comment by LeavesBreathe** [comment]
Currently in the translate model, they take the argmax of these logits for the next word. The words aggregated together give the predicted sentence.
